free-3.md

```diff
@@ -57,12 +75,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think that a decent amount of the project would involve translating between the various social media bot API services and the GUI of this language. That could involve a direction translation with many similar functions or it could involve clever software design and a lot of pattern matching. Depending on the implementation, the time allocations for the project could vary. The similarities between different social media APIs would determine how much of the project would involve writing functions specific to one type of bot.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project has the potential to scale out of control very quickly. It would be easiest to start with one social network and gradually add more. At this level, it's a fairly ambitious project but not too overwhelming. It would allow the developer to become familiar with bot creation and make a lot of decisions about user accessibility.
+
```

Comment:
I think this is also a very ambitious project. The example you provided kind of shows how wide the scope of this project can be (considering a company is able to make a living off of this). To me, it feels like limiting to one social network and focusing on either bot creation or bot execution would be more realistic. Perhaps even only selected a few of the aforementioned features.